src/emulator/commandUtils.ts

```diff
@@ -29,22 +31,27 @@ export async function beforeEmulatorCommand(options: any): Promise<any> {
     config: DEFAULT_CONFIG,
   };
   const optionsWithConfig = options.config ? options : optionsWithDefaultConfig;
-
-  // TODO(samstern): Do we still want to do this?
-  const requiresAuth = controller.shouldStart(optionsWithConfig, Emulators.HOSTING);
-
   const canStartWithoutConfig =
     options.only &&
     !controller.shouldStart(optionsWithConfig, Emulators.FUNCTIONS) &&
     !controller.shouldStart(optionsWithConfig, Emulators.HOSTING);
 
+  try {
+    requireAuth();
```

Comment:
Should this instead be `await requireAuth(options);`?

Reply:
I tried to make this change locally, and instead of getting the output in https://github.com/firebase/firebase-tools/issues/1854#issuecomment-565620230, I get this instead:

```
kamik@RED-X1C6 MINGW64 /d/sandbox/firebase-emulator (master)
$ npx firebase emulators:start --only hosting 
!  emulators: You are not currently authenticated so some features may not work correctly. Please run firebase login to authenticate the CLI.
i  emulators: Starting emulators: hosting
!  hosting: Could not fetch web app configuration and there is no cached configuration on this machine. Check your internet connection and make sure you are authenticated. To continue, 
you must call firebase.initializeApp({...}) in your code before using Firebase.
i  hosting: Serving hosting files from: public
+  hosting: Local server: http://localhost:5000
+  hosting: Emulator started at http://localhost:5000
+  All emulators started, it is now safe to connect.
```

Reply:
D'oh thank you!  Yes it should.